x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -293,16 +293,14 @@ export class HeadlessChromiumDriverFactory {
    */
   private async getErrorMessage(message: ConsoleMessage): Promise<undefined | string> {
     for (const arg of message.args()) {
-      const errorMessage = await arg
-        .executionContext()
-        .evaluate<undefined | string>((_arg: unknown) => {
-          /* !! We are now in the browser context !! */
-          if (_arg instanceof Error) {
-            return _arg.message;
-          }
-          return undefined;
-          /* !! End of browser context !! */
-        }, arg);
+      const errorMessage = await arg.executionContext().evaluate((_arg: unknown) => {
```

Comment:
Looks like we don't use the `.evaluateContext()` part any longer:
https://github.com/puppeteer/puppeteer/blob/v18.0.2/docs/api/puppeteer.jshandle.evaluate.md

```suggestion
      const errorMessage = await arg.evaluate((_arg: unknown) => {
```